src/main/java/HTTPVerifierInterface.java

```diff
@@ -0,0 +1,7 @@
+package com.td.HTTPServer;
+
+public interface HTTPVerifierInterface {
```

Comment:
I'm not a big fan of using `Interface` (or variants—sometimes you'll just see an `i`) in the name of an interface. In a situation like this, for example, it seems like you're just making an interface for the sake of making one and nothing else. What do you think of something like `interface Verifier` or `interface Validator` and then one implementation can be an `HttpVerifier`/`HttpValidator`?